app/src/main/java/com/mobven/shortly/executor/PostExecutionThreadImpl.kt

```diff
@@ -0,0 +1,14 @@
+package com.mobven.shortly.executor
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import javax.inject.Inject
+
+class PostExecutionThreadImpl @Inject constructor() : PostExecutorThread {
```

Comment:
Ravzacım bunu neden yazdık tam olarak, testini yazmak için mi yoksa başka bir şey için mi?

Reply:
Dispatcher lar için eklendi Esracım, UpdateFavoriteUseCase ve ShortenLinkUseCase e  ekledim.